R/resp.R

```diff
@@ -81,7 +81,7 @@ new_response <- function(method,
   headers <- as_headers(headers, error_call = error_call)
   # ensure we always have a date field
   if (!"date" %in% tolower(names(headers))) {
-    headers$Date <- http_date()
```

Comment:
@mgirlich FYI. I think it's the right decision to do it here since `response()` is overwhelming used in tests/mocking.